### PR TITLE
I#738: Add basic `Sentry` setup to Miru-Web

### PR DESCRIPTION
**Pull request by nisusam**
## Summary
- Fix https://github.com/saeloun/miru-web/issues/738
- Add the `Sentry` gem and update the configuration setting
- Remove the `Honeybadger` gem
- @akhilgkrishnan Please add the `SENTRY_DSN` environment variable to respective environments

## Preview

Screenshot 2022-10-16 at 11 18 52 PM

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## How Has This Been Tested?

- I have set up an individual 14-day trial account on sentry
- Created rails project & got the DSN value
- Added the same in the `.env` file to test the same
- Added the following code-block to index action of `InternalApi::V1::TeamController` class

```RUBY
begin
  1 / 0
rescue ZeroDivisionError => exception
  Sentry.capture_exception(exception)
end
```

- Now, after hitting the `Team` page, it throws an exception
- This exception message is captured in sentry as follows

Screenshot 2022-10-16 at 11 18 52 PM

### Checklist:

- [x] I have manually tested all workflows
- [x] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -45,3 +45,6 @@ WISE_PROFILE_ID="123456"
 
 #Feature Flags
 ENABLE_WEEKLY_REMINDER=
+
+# Sentry
+SENTRY_DSN =
```
Comment (keshavbiswa) [inline .env.example:50]:
```suggestion
SENTRY_DSN=
```

**Review by keshavbiswa — COMMENTED** [review]
Added my comments

**Review by akhilgkrishnan — APPROVED** [review]
LGTM

File: config/initializers/sentry.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+  # Set traces_sample_rate to 1.0 to capture 100%
+  # of transactions for performance monitoring.
+  # We recommend adjusting this value in production.
+  config.traces_sample_rate = 1.0
+
+  # Excluding Exceptions
+  config.excluded_exceptions += ["ActionController::RoutingError"]
+
+  config.enabled_environments = %w[development production]
```
[the remark this answers is not in the document]
Comment (akhilgkrishnan) [reply config/initializers/sentry.rb:18]: i think we can remove development

**Pull request merged**